packages/mux-player/test/player.test.js

```diff
@@ -216,37 +188,75 @@ describe('<mux-player>', () => {
     assert.equal(muxVideo.preload, defaultPreload, `muxVideo default preload is ${defaultPreload}`);
   });
 
-  it('poster is forwarded to the media element', async function () {
+  it('poster is forwarded to the media-poster-image element', async function () {
     const player = await fixture(`<mux-player
       poster="https://image.mux.com/xLGf7y8cRquv7QXoDB02zEe6centwKfVmUOiPSY02JhCE/thumbnail.jpg?time=0"
     ></mux-player>`);
-    const muxVideo = player.media;
+    const mediaPosterImage = player.theme.shadowRoot.querySelector('media-poster-image');
 
     assert.equal(
       player.poster,
-      'https://image.mux.com/xLGf7y8cRquv7QXoDB02zEe6centwKfVmUOiPSY02JhCE/thumbnail.jpg?time=0'
+      'https://image.mux.com/xLGf7y8cRquv7QXoDB02zEe6centwKfVmUOiPSY02JhCE/thumbnail.jpg?time=0',
+      'player.poster does not equal in-html defined poster attribute'
     );
     assert.equal(
-      muxVideo.poster,
-      'https://image.mux.com/xLGf7y8cRquv7QXoDB02zEe6centwKfVmUOiPSY02JhCE/thumbnail.jpg?time=0'
+      mediaPosterImage.getAttribute('src'),
+      'https://image.mux.com/xLGf7y8cRquv7QXoDB02zEe6centwKfVmUOiPSY02JhCE/thumbnail.jpg?time=0',
+      'mediaPosterImage src attribute does not equal in-html defined poster attribute'
     );
 
     player.removeAttribute('poster');
-    assert(!muxVideo.hasAttribute('poster'), `has poster attr removed`);
+    assert(!mediaPosterImage.hasAttribute('src'), `has src attr removed`);
 
     player.setAttribute(
       'poster',
       'https://image.mux.com/xLGf7y8cRquv7QXoDB02zEe6centwKfVmUOiPSY02JhCE/thumbnail.jpg?time=1'
     );
     assert.equal(
-      muxVideo.getAttribute('poster'),
+      mediaPosterImage.getAttribute('src'),
       'https://image.mux.com/xLGf7y8cRquv7QXoDB02zEe6centwKfVmUOiPSY02JhCE/thumbnail.jpg?time=1',
       `has poster attr added`
     );
+  });
+
+  it('poster can be unset with an empty string', async function () {
```

Comment:
I thought it gets unset with `null`?

Reply:
we can't use `null` because it means absence of poster attribute which means the poster will be derived from the playback-id

Reply:
aah, right